src/api/app/mixins/webui/manage_relationships.rb

```diff
@@ -29,6 +19,10 @@ def save_person_or_group(what)
     end
   end
 
+  def users_path
```

Comment:
Please rename this to something else to not overshadow the path helper method with the same name.

Reply:
done